packages/voter-dapp/src/ResolvedRequests.js

```diff
@@ -250,13 +250,15 @@ function ResolvedRequests({ votingAccount }) {
                 <TableCell>{hexToUtf8(resolutionData.identifier)}</TableCell>
                 <TableCell>{formatDate(resolutionData.time, web3)}</TableCell>
                 <TableCell>Resolved</TableCell>
-                <TableCell>{isAdminVote ? translateAdminVote(userVote) : userVote}</TableCell>
+                <TableCell>{isAdminVote && userVote !== "No Vote" ? translateAdminVote(userVote) : userVote}</TableCell>
```

Comment:
why this change?

Reply:
Becuase otherwise an Admin vote will call `translateAdminVote("No Vote")` which returns "INVALID ADMIN VOTE". This is the easiest fix for that